src/hubverse_infrastructure/shared/hubverse_transforms.py

```diff
@@ -210,16 +210,19 @@ def create_lambda_package_placeholder(s3_bucket: str, s3_key: str):
         raise Exception(f"Error when checking for existing lambda package: {s3_bucket}/{s3_key}") from e
 
 
-def create_transform_infrastructure():
+def create_transform_infrastructure() -> tuple[aws.lambda_.Function, aws.iam.Role]:
```

Comment:
No substantive change to this logic...just having this function return the lambda function and role so they can be used when provisioning individual hub resources.

Reply:
Interesting you use a tuple here instead of just a list... Personal preference?

Reply:
tbh, it's because I didn't know you could unpack a list in the same way you can unpack a tuple until this question made me google it 😄 

but yeah, I do like tuples!